### ISSUE TITLE
pyflakes3 ?

### ISSUE THREAD

**Issue by AlexDaniel** [Wilfred/pyimport#11]
The documentation says:

> This requires pyflakes to be on PATH. Alternatively, set pyimport-pyflakes-path.

However, debian unstable seems to only provide pyflakes3, not pyflakes. Shouldn't pyimport automatically attempt to use it?

**Comment by xaperret** [comment]
got the same problem, did you managed to find a way to solve it ?

**Comment by AlexDaniel** [comment]
Yes! Actually, it works now. I have no idea why. Something, somewhere, fixed it.
